core-bundle/src/Twig/Global/ContaoVariable.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Contao\CoreBundle\Twig\Global;
+
+use Contao\CoreBundle\Csrf\ContaoCsrfTokenManager;
+use Contao\CoreBundle\Security\Authentication\Token\TokenChecker;
+use Contao\PageModel;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class ContaoVariable
+{
+    public function __construct(
+        private readonly RequestStack $requestStack,
+        private readonly TokenChecker $tokenChecker,
+        private readonly ContaoCsrfTokenManager $tokenManager,
+    ) {
+    }
+
+    public function getPage(): PageModel|null
+    {
+        $pageModel = $this->requestStack->getCurrentRequest()?->attributes->get('pageModel');
+
+        if ($pageModel instanceof PageModel) {
+            return null;
+        }
+
+        return $pageModel;
```

Comment:
This looks wrong to me.

```suggestion
        if (!$pageModel instanceof PageModel) {
            return null;
        }

        return $pageModel;
```

Reply:
Fixed in 2a215a11fd4e61b0351bd38efdcb4b674cf2e161.